tools/Godot.AppDir/AppRun

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+HERE=$(dirname $(readlink -f "${0}"))
```

Comment:
Always be careful with bash, this doesn't work for folders with spaces:

![image](https://cloud.githubusercontent.com/assets/1072168/18184283/f539cc2e-70a0-11e6-9076-2d8384335b13.png)

The correct command:

```bash
HERE="$(dirname "$(readlink -f "${0}")")"
```